CSV/sample-entries.csv

```diff
@@ -20,9 +20,12 @@ dtse,"DTS Express low bit rate audio, also known as DTS LBR",Audio,DTS,0xAC
 dtsh,DTS-HD High Resolution Audio,Audio,DTS,0xAA
 dtsl,DTS-HD Master Audio,Audio,DTS,0xAB
 dtsx,DTS:X,Audio,DTS,
-dvav,AVC-based “Dolby Vision”,Video,Dolby Vision,
-dvhe,HEVC-based “Dolby Vision”,Video,Dolby Vision,
+dva1,AVC-based “Dolby Vision” derived from 'avc1',Video,Dolby Vision,
```

Comment:
The last value should be `dolbyvision` as this is the 'internal' link name (not user visible). Same for the other 3 lines.